infrastructure/develop/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3"
+
+services:
+  postgres:
+    image: postgres:latest
```

Comment:
we should use same version as in prod... idk what prod will use, but koh uses 11.5 if we wanna use that

Reply:
totally!!